aiohttp/web_server.py

```diff
@@ -53,17 +54,21 @@ def handle_request(self, message, payload):
 
         request = self._request_factory(message, payload, self)
         self._request = request
+        resp = None
 
         try:
             try:
                 resp = yield from self._handler(request)
             except HTTPException as exc:
                 resp = exc
+            except HttpProcessingError as exc:
```

Comment:
All exceptions not derived from `HttpProcessingError` should raise `500 Internal Server Errror` (now it's done by `server.py` code).